docs: multiline export (#47925)
metabase/metabase: docs/installation-and-operation/serialization.md

### Example of a serialized question

Questions can be found in the `cards` directory of a collection directory. Here's an example card YAML file for a question written with SQL that uses a field filter and has an area chart visualization.

> > If your native query has trailing whitespace, YAML will convert your query to a single string literal (this only affects presentation, not functionality). Remove trailing whitespace from native queries to keep multi-line format.

Reviewer: ```suggestion
> To preserve a native query's multi-line format, remove trailing whitespace from native queries. If your native query has trailing whitespace, YAML will convert your query to a single string literal (which only affects presentation, not functionality). 
```
